tests/integration/priority.js

```diff
@@ -0,0 +1,152 @@
+import expect from 'expect.js'
```

Comment:
I put all tests which touch the dom functional. Normally I try to separate integration and functional tests.